packages/devnet/run.sh

```diff
@@ -35,7 +35,9 @@ sleep 10
 # to break when there are fewer than 3 blocks in the chain. (We have no idea
 # why, but it was consistently reproducible).
 /geth --datadir node0/ attach --exec 'eth.sendTransaction({"from": "0x5409ED021D9299bf6814279A6A1411A7e866A631", "to": "0x84bd1cfa409cb0bb9b23b8b1a33515b4ac00a0af", "value": "0x1"})'
+sleep 2
```

Comment:
I had to add this extra sleep in order for the tests to work on CI. Without it, all 3 transactions ended up in the same block, making it so there is only 1 block in the chain (besides the genesis block). Who knows why, but our tests fail unless the blockchain has at least 3 blocks.